src/lib/supabase.ts

```diff
@@ -0,0 +1,72 @@
+import fp from "fastify-plugin";
+import { FastifyInstance } from "fastify";
+
+import { createClient, SupabaseClient } from "@supabase/supabase-js";
+declare module "fastify" {
+  export interface FastifyInstance {
+    supabase: SupabaseClient;
+  }
+  // interface FastifyRequest {
+  //   usersPluginProp: string;
+  // }
+  // interface FastifyReply {
+  //   usersPluginProp: number;
+  // }
+}
+
+export interface SupabasePluginOptions {
+  supabaseUrl: string;
+  supabaseServiceRoleKey: string;
+}
+
+const FastifySupabase = async (
+  fastify: FastifyInstance,
+  opts: SupabasePluginOptions
+) => {
+  const supabaseAdmin = createClient(
+    opts.supabaseUrl,
+    opts.supabaseServiceRoleKey,
+    {
+      autoRefreshToken: false,
+      persistSession: false,
+      detectSessionInUrl: false,
+      realtime: {
+        timeout: 1,
+      },
+    }
+  );
+  if (!fastify.supabase) {
+    fastify.decorate("supabase", supabaseAdmin);
+  }
+};
+
+// async function main(): Promise<void> {
```

Comment:
What is this commented code?

Reply:
Thanks. Was a left over from the external repo.